SR2019/2019-10-16-soton-kickstart-deny.md

```diff
@@ -0,0 +1,13 @@
+---
+to: Student Robotics 2019, Southampton Segment
+subject: The London kickstart is full
+note: This email is to be sent to those who are _not_ going to the London kickstart.
+---
+
+Hello!
+
+We regret to update you that the London kickstart venue is full, and so your team will not be able to go.
+
+There is space for you at the Southampton venue, so we'd highly suggest going there instead. If Southampton is not viable for you, then we can arrange another way to get you your kit.
```

Comment:
Maybe expand on 'viable', like 'If Southampton is too hard for you to get to, then we can..."